bbb-learning-dashboard/src/components/QuizzesTable.jsx

```
@@ -0,0 +1,462 @@
+
+    anonGridCols.push({
+      ...commonColProps,
+      sortable: false,
+      renderCell: (params) => <GridCellExpand value={params?.value || ''} width={params?.colDef?.computedWidth} />,
+    });
+
+    gridCols.push({
+      ...commonColProps,
+      sortable: true,
+      valueGetter: (params) => {
+        const [isCorrect] = params?.value;
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)